sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -835,21 +835,11 @@ case class DescribeColumnCommand(
 case class ShowTablesCommand(
     databaseName: Option[String],
     tableIdentifierPattern: Option[String],
+    override val output: Seq[Attribute] = Seq.empty,
     isExtended: Boolean = false,
     partitionSpec: Option[TablePartitionSpec] = None) extends RunnableCommand {
 
-  // The result of SHOW TABLES/SHOW TABLE has three basic columns: database, tableName and
-  // isTemporary. If `isExtended` is true, append column `information` to the output columns.
-  override val output: Seq[Attribute] = {
-    val tableExtendedInfo = if (isExtended) {
-      AttributeReference("information", StringType, nullable = false)() :: Nil
-    } else {
```

Comment:
Seems you missed information column?

Reply:
The output should be delivered by logical nodes(ShowTables and ShowTableExtended) in normal.

Reply:
> The output should be delivered by logical nodes(ShowTables and ShowTableExtended) in normal.

Yea, got it， thanks.